src/sql/workbench/browser/modal/modal.ts

```diff
@@ -175,8 +177,8 @@ export abstract class Modal extends Disposable implements IThemable {
 			builderClass += ' flyout-dialog';
 		}
 
-		if (this._modalOptions.isWide) {
-			builderClass += ' wide';
+		if (this._modalOptions.width && typeof this._modalOptions.width === 'string') {
```

Comment:
Probably easier to just  set the width here for all options and just switch on the different possible values instead of using css rules for some and  specifically setting the width for others